extensions/applicationinsights-cfgsync-js/package.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "@microsoft/applicationinsights-cfgsync-js",
+    "version": "0.0.1",
+    "description": "Microsoft Application Insights CfgSync plugin",
+    "homepage": "https://github.com/microsoft/ApplicationInsights-JS#readme",
+    "author": "Microsoft Application Insights Team",
+    "main": "dist/applicationinsights-cfgsync-js.js",
+    "module": "dist-esm/applicationinsights-cfgsync-js.js",
+    "types": "types/applicationinsights-cfgsync-js.d.ts",
+    "sideEffects": false,
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/ApplicationInsights-JS/tree/master/extensions/applicationinsights-cfgsync-js"
+    },
+    "scripts": {
+        "clean": "rm -rfv browser types dist-esm",
+        "build": "npm run build:esm && npm run build:browser && npm run sri && npm run dtsgen",
+        "build:esm": "grunt cfgsync",
+        "build:browser": "rollup -c rollup.config.js --bundleConfigAsCjs",
+        "rebuild": "npm run build",
+        "test": "grunt cfgsynctests",
+        "mintest": "grunt cfgsync-mintests",
+        "lint": "tslint -p tsconfig.json",
+        "dtsgen": "api-extractor run --local && node ../../scripts/dtsgen.js \"Microsoft Application Insights idsync plugin\"",
+        "sri": "node ../../tools/subResourceIntegrity/generateIntegrityFile.js",
+        "ai-min": "grunt cfgsync-min",
+        "ai-restore": "grunt cfgsync-restore",
+        "npm-pack": "npm pack"
+    },
+    "devDependencies": {
+        "@nevware21/ts-async": ">= 0.1.0 < 2.x",
```

Comment:
We can't use this either as it's ES5+ and master needs to support ES3, especially for this case.

Reply:
I added it for testing only! it won't be used in any non-testing files!

Reply:
Here is another one for dependency plugin and it is also used for testing https://github.com/microsoft/ApplicationInsights-JS/blob/3a9fc30c73c99f394af3817b9c8c88fa937bed68/extensions/applicationinsights-dependencies-js/package.json#L30